neo/SmartContract/ApplicationEngine.cs

```diff
@@ -92,7 +92,8 @@ protected override bool PreExecuteInstruction()
                 Version = 0,
                 PrevHash = snapshot.CurrentBlockHash,
                 MerkleRoot = new UInt256(),
-                Timestamp = snapshot.Blocks[snapshot.CurrentBlockHash].Timestamp + Blockchain.SecondsPerBlock,
+                // TODO - Check this sum
+                Timestamp = snapshot.Blocks[snapshot.CurrentBlockHash].Timestamp + Blockchain.MillisecondsPerBlock,
```

Comment:
What do you want to achieve with this?  The sum looks correct to me. 
Previous Block Timestamp (in MS) + Milliseconds per block = New "Block" expected timestamp.

Reply:
I do not know. Can you check @shargon?

Reply:
Let's rename this to TimestampMS. Timestamp is seconds from unix epoch, we need a different name, or confusions will arise.
I don't like `MillisecondsPerBlock`, let's please use `TimePerBlock` (and document it's milliseconds).

Reply:
`MillisecondsPerBlock` is uint, and should be `ulong` but yes, the sum is good